app/api/services/insight.py

```diff
@@ -0,0 +1,50 @@
+import pendulum
+from app.api.helpers import Service
+from app.models import Insight, db
+
+
+class InsightService(Service):
+    __model__ = Insight
+
+    def __init__(self, *args, **kwargs):
+        super(InsightService, self).__init__(*args, **kwargs)
+
+    def get_insight(self, now, active_only=None):
+        query = (
+            db
+            .session
+            .query(
+                Insight.id,
+                Insight.data
+            )
+        )
+
+        if active_only is True or active_only is None:
```

Comment:
```suggestion
        if active_only:
```
To go along with the change in default value.